src/main/webapp/IFrame.js

```diff
@@ -151,6 +151,7 @@ function resetPlaybackInfo() {
     timestamp: 0,
     videoSpeed: 1,
     videoId: '',
+    skipVotes: 0,
```

Comment:
can to name it as votesToSkipVideo. Skip votes sounds like we want to skip voting.